docs/Quick-start-and-introduction.md

```diff
@@ -149,7 +149,7 @@ The result sets objects code returned from `getCode()` will match one of these c
  * `'FAILURE'`, value of `Result::FAILURE`, on any general failure.
  * `'FAILURE_IDENTITY_NOT_FOUND'`, value of `Result::FAILURE_IDENTITY_NOT_FOUND`, when identify could not be found.
  * `'FAILURE_CREDENTIALS_INVALID'`, value of `Result::FAILURE_CREDENTIALS_INVALID`, when credentials are invalid.
- * `'FAILURE_CREDENTIALS_NOT_FOUND_IN_REQUEST_IN_REQUEST'`, value of `Result::FAILURE_CREDENTIALS_NOT_FOUND_IN_REQUEST`, when credentials are not found in request.
+ * `'FAILURE_CREDENTIALS_NOT_IN_REQUEST_IN_REQUEST'`, value of `Result::FAILURE_CREDENTIALS_NOT_IN_REQUEST`, when credentials are not found in request.
```

Comment:
..._IN_REQUEST_IN_REQUEST?

Reply:
bugs in my brain... eh find and replace ;-)? (fixed already)